Tasks/ANT/task.json

```diff
@@ -44,6 +44,7 @@
     "inputs": [
         {
             "name": "antBuildFile",
+            "aliases": ["buildXml"],
```

Comment:
In my opinion the "buildXml" alias departs too much from the label "Ant build file" and so it will be difficult for customers to associate the UI name with the YAML name.

Reply:
Remove entirely and leave with just `antBuildFile` or have `buildFile` too?

Reply:
I like having one alias called "buildFile".

Reply:
buildFile is also the actual argument we pass to ant command line 👍